src/TestReactors.jl

```diff
@@ -194,6 +194,8 @@ using Sundials
         sol = solve(react.ode, CVODE_BDF(), abstol=1e-20, reltol=1e-12) #solve the ode associated with the reactor
         sim = Simulation(sol, domain)
 
+        rops(sim, "H2", 220.44002454)
```

Comment:
this just tests that this runs...shouldn't we test the output value?...otherwise we wouldn't catch off by one bugs...

Reply:
Good catch! I have added a test to check the output value.